Setttings: strip whitespace between entries

fliiiix opened the pull request:
It took me a lot of time to figure out why this config does not do what I would expect:

```
CSRF_TRUSTED_ORIGINS = "https://foo.bar.example/, http://foo.example/"
```

The first commit 640000366a2b47f4fecbc9c3337a53c1d028de22  addresses this and strips white space.
I feel like refactoring that out into helper functions a4e34acd6e32e64f0802ad19a47dd6b318a62411 makes it more readable, but i can also remove that commit if thats not in line with the project.
And i used black for formatting 03c10bc1b1807a9e39ed9b2f399a9aa4901b23ab since i wasn't sure what the policy is here, but also in a extra commit which can be removed easily again.

I feel all these changes are useful but let me know :D

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/TandoorRecipes/recipes?pullRequest=3161) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/TandoorRecipes/recipes?pullRequest=3161) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/TandoorRecipes/recipes?pullRequest=3161) it.</sub>

vabene1111 commented:
Hi, thank you for the PR, I am very busy atm but will try to look into this as soon as possible.

vabene1111 commented:
ok so sorry for the long delay. The logic change looks good. May I ask you to do two things: 

1. revert the formatting, It was well intended but formatting is a very difficult topic that has not yet been 100% decided
2. can you change the branch to merge into feature/vue3, because I don't want to make any setting changes on develop any more

thank you

fliiiix commented:
> revert the formatting, It was well intended but formatting is a very difficult topic that has not yet been 100% decided

I can highly recommend using black and stop discussing about it, worked well for me at work and in my open source projects,
but i removed it from this MR 

> can you change the branch to merge into feature/vue3
rebased on new branch

let me know if there is anything else required to get it merged :)

vabene1111 commented:
thank you. I know black works very well but my personal preference is in the way, I just simply dont like how it looks.
